### DISCUSSION TITLE
New version bump

### DISCUSSION THREAD

**Discussion by thetonus** [mindee/doctr#1277] [General]
When will there be a new version bump? There have been many changes since 0.6.0 was released. There are some new features I want to use, but I do not want to set poetry to track the master branch (since that is not deterministic).

**Comment by felixdittrich92** [comment]
Hi @thetonus 👋,
I agree the last release is some time away 🙈
We are ready for the next release but will wait until #969 is finished that each model has a pretrained checkpoint.
For visibility @odulcy-mindee

**Reply by felixdittrich92** [reply]
I think this ticket will be finished in october / november

**Reply by odulcy-mindee** [reply]
Hello @thetonus,

As @felixdittrich92 said, we are in the process of training the models that have no weight. October / November is a good estimate

**Reply by thetonus** [reply]
Thanks for the quick reply. Sounds good.
